### ISSUE TITLE
Recommend Editor/IDE

### ISSUE THREAD

**Issue by choiwhmarco** [open-learning-exchange/open-learning-exchange.github.io#2777]
### Problem
There is no recommend editor/IDE for coding, so I would want to add some links for recommendation.
### Screenshots

![Capture](https://user-images.githubusercontent.com/22423089/66602164-624a4b00-eb5e-11e9-89f1-ad2436191da4.PNG)

### Proposed solution

Add links for  [Sublime](https://www.sublimetext.com/), [Visual Studio](https://code.visualstudio.com/), [WebStorm](https://www.jetbrains.com/webstorm/?fromMenu)

**Comment by sjkadali** [comment]
This would really be helpful for interns in choosing the recommended editor and getting more info about it.

**Comment by sauravghosal** [comment]
Yup! I believe that having a recommended code editor/IDE is important to help interns prepare for the internship.
